pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -57,8 +57,10 @@ type KubernetesMetaConfiguration struct {
 	// TODO: use a proper toleration structure?
 	PodToleration map[string]string `json:"toleration,omitempty"`
 	// TODO: use namespacedname
-	PodEnvironmentConfigMap string `json:"pod_environment_configmap,omitempty"`
-	PodPriorityClassName    string `json:"pod_priority_class_name,omitempty"`
+	PodEnvironmentConfigMap  string            `json:"pod_environment_configmap,omitempty"`
+	PodEnvironmentSecretName string            `json:"pod_environment_secret_name,omitempty"`
+	PodEnvironmentSecretKeys map[string]string `json:"pod_environment_secret_keys,omitempty"`
+	PodPriorityClassName     string            `json:"pod_priority_class_name,omitempty"`
```

Comment:
This means there will be a single k8s secret with env vars shared between all pods of PG clusters. In the case of `minio` this is valid behavior, otherwise one would need to setup access control. This needs at least some documentation

Reply:
That's consistent with behaviour of `PodEnvironmentConfigMap` - a single ConfigMap that's shared between all pods.

I think of `PodEnvironmentSecretName`/`PodEnvironmentSecretKeys` as of secret version of `PodEnvironmentConfigMap`.